docs/conf/portland/2020/opportunity-grants.rst

```diff
@@ -0,0 +1,57 @@
+:template: {{year}}/generic.html
+
+Opportunity grants
+==================
+
+The grant program for WTD Portland 2020 supports people who would otherwise not be able to attend the conference by providing free tickets and covering travel and accommodation costs. Applications are open to anyone who wants to attend Write the Docs.
```

Comment:
```suggestion
The grant program for WTD Portland 2020 supports people who would otherwise not be able to attend the conference by funding for any related expenses of attending. Applications are open to anyone who wants to attend Write the Docs.
```